freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/keytab/model/ServiceKeytabRequest.java

```diff
@@ -0,0 +1,62 @@
+package com.sequenceiq.freeipa.api.v1.keytab.model;
+
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.sequenceiq.freeipa.api.v1.keytab.doc.KeytabModelDescription;
+import com.sequenceiq.service.api.doc.ModelDescriptions;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel("ServiceKeytabV1Request")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class ServiceKeytabRequest {
+
+    @ApiModelProperty(KeytabModelDescription.ID)
+    private Long id;
+
+    @ApiModelProperty(value = ModelDescriptions.ENVIRONMENT_CRN, required = true)
+    @NotNull
+    private String environmentCrn;
+
+    @ApiModelProperty(value = KeytabModelDescription.SERVICE_NAME, required = true)
+    @NotNull
+    private String serviceName;
+
+    @ApiModelProperty(value = KeytabModelDescription.SERVICE_HOST, required = true)
+    @NotNull
+    private String serverHostName;
+
+    public Long getId() {
```

Comment:
I am not sure if you have used lombok before. It doesn't look like that is used in the cloudbreak repo yet (but it is in thunderhead). You might be able to use lombok to implement the getter/setters automatically. I just thought it might cut down on the boilerplate code.

If you do implement it here, you may want to do it for the other getters/setters in this commit. But it may be better to just be consistent with the rest of this repo and thus ignore this idea.

Reply:
I know Lombok, but cloudbreak doesn't use it. Let's follow the current way of doing things.

Reply:
ok